index.js

```diff
@@ -4,7 +4,49 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+//const parameters = process.argv
+ let i;
+ let result = 0;
```

Comment:
We want to avoid global variables. The `i` variable is being accessed by every for loop below. While it didn't cause an error. It can make things complicated in the future when you have more complex code. The `i` variable for you for loop can be declared within the parenthesis of the for loop. Check out this [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Loops_and_iteration#for_statement). 

your result variable can be declared in the function variable and made accessible throughout the rest of the function